benchmarks/appsec_iast_propagation/scenario.py

```diff
@@ -2,12 +2,12 @@
 
 import bm
 
+from ddtrace.appsec.iast._taint_tracking import OriginType
+from ddtrace.appsec.iast._taint_tracking import Source
+from ddtrace.appsec.iast._taint_tracking import TaintRange
```

Comment:
I'm confused what this reordering of imports does to the benchmark. Can you clarify?

Reply:
Sorry, realized the package namespace has changed.